README.md

```diff
@@ -45,6 +45,26 @@ If you use GLASS simulations or the GLASS library in your research, please
 [cite the original GLASS paper](https://glass.readthedocs.io/stable/user/publications.html)
 in your publications.
 
+    @ARTICLE{2023OJAp....6E..11T,
+        author = {{Tessore}, Nicolas and {Loureiro}, Arthur and
+        {Joachimi}, Benjamin and {von Wietersheim-Kramsta}, Maximilian
```

Comment:
Should this be in a single line and ignore the linter rule?

Reply:
Yes, I think we'll have to https://prettier.io/docs/en/ignore.html#markdown